README.md

```diff
@@ -31,12 +31,12 @@ Add the following lines to your Podfile:
 
 ```ruby
 # Required
-pod 'MapboxNavigation.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :commit => 'c5a637a201a3043c24e52f3e2c50a1657a8ffced'
-pod 'MapboxNavigationUI.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :commit => 'c5a637a201a3043c24e52f3e2c50a1657a8ffced'
-pod 'OSRMTextInstructions', :git => 'git@github.com:Project-OSRM/osrm-text-instructions.swift.git', :commit => 'bdd962bf5593f951bbbbd26435f732a829dacda7'
+pod 'MapboxNavigation.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :tag => 'v0.1.0'
+pod 'MapboxNavigationUI.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :tag => 'v0.1.0'
+pod 'OSRMTextInstructions', :git 'https://github.com/Project-OSRM/osrm-text-instructions.swift.git' => , :tag => 'v0.1.0'
```

Comment:
I just pushed v0.1.0 to CocoaPods.org, so you should remove this line and place the following line in the podspec:

```rb
s.dependency 'OSRMTextInstructions', '~> 0.1'
```